### PR TITLE
Update to setup section

### PR DESCRIPTION
**Pull request by MelwynPaul**
Renamed topic title as per ToC reorg - 'Use a private registry' to 'Install in a Disconnected Environment'

### CODE REVIEW

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ---
-title: Use a Private Registry
-description: Set up Verrazzano using a private container registry
+title: Install in a Disconnected Environment
+description: Learn how to install Verrazzano in a disconnected environment
```
Comment (rosemarymarano) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:3]: -> description: Set up Verrazzano using a private container registry
(I think that we need to keep a connection to the original Use a Private Registry doc.) 
Also, you should probably invite the developer who wrote the original doc and perhaps, the ones that worked on Air Gap to make sure that we are on the right tract by renaming this doc.

**Review by fredtibbitts — COMMENTED** [review]
I don't have a strong opinion on the title change. I guess my only hesitation is that users can install from a private registry and not be in a disconnected environment, although disconnected is the main use case for private registry.

**Comment by rosemarymarano** [comment]
@fredtibbitts Thank you, I think that's an important distinction.
@MelwynPaul Maybe we should consider keeping the original title and make note of its main use case, installing while in a disconnected environment.

**Comment by MelwynPaul** [comment]
Thank you, @rosemarymarano 
Thank you, @fredtibbitts 

@fredtibbitts  - At the beginning of the topic we added - "You may chose to install Verrazzano from a private registry, particularly if you are in a disconnected environment." 
Is that accurate and sufficient?

**Review by rosemarymarano — COMMENTED** [review]
minor edit

File: content/en/docs/setup/private-registry/private-registry-full-distribution.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ Weight: 4
 draft: false
 ---
 
+You may chose to install Verrazzano from a private registry, particularly if you are in a disconnected environment.
```
Comment (fredtibbitts) [inline content/en/docs/setup/private-registry/private-registry-full-distribution.md:8]: chose -> choose

This wording is a little bit awkward I think. If the user is running in a disconnected environment then they don't have a choice; they have to use a private registry.

Comment (rosemarymarano) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:8]: @fredtibbitts How is this suggested rewording? 
**NOTE**: If you are running in a disconnected environment (without access to the Internet), then you will need to install Verrazzano using a private container registry.

Comment (MelwynPaul) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:8]: @fredtibbitts - Need your help. 
Do you agree to add the above reworded note?

Comment (fredtibbitts) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:8]: That wording seems fine to me.

Comment (MelwynPaul) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md:8]: Thank you, @fredtibbitts.

Comment (MelwynPaul) [reply content/en/docs/setup/private-registry/private-registry-full-distribution.md]: Thank you, @fredtibbitts.

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**